packages/Adapters/POD_C/src/DTK_C_API.h

```diff
@@ -0,0 +1,49 @@
+#ifndef DTK_NONAME_H
+#define DTK_NONAME_H
+
+#include "mpi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// QUESTION: Do we want to be able to track missed entities?
+
+// Convenience typedef for an opaque pointer
+typedef void DTK_Map;
```

Comment:
Not sure I understand the question.
I did not do
```
typedef void* DTK_Map;
```
so that user sees he is dealing with a pointer and understand he is responsible for calling delete.